aws_lambda_powertools/utilities/validation/validator.py

```diff
@@ -44,6 +47,14 @@ def validator(
         Custom formats containing a key (e.g. int64) and a value expressed as regex or callback returning bool
     outbound_formats: Dict
         Custom formats containing a key (e.g. int64) and a value expressed as regex or callback returning bool
+    inbound_handlers: Dict
+        Custom methods to retrieve remote schemes, keyed off of URI scheme
+    outbound_handlers: Dict
+        Custom methods to retrieve remote schemes, keyed off of URI scheme
+    inbound_provider_options: Dict
+        Arguments that will be passed directly to the underlying validate call for the inbound event
+    outbound_provider_options: Dict
+        Arguments that will be passed directly to the underlying validate call for the outbound event
```

Comment:
genuine question - is there a reason why we can't rename `**kwargs` to `**provider_options` instead of adding new ones?

More options typically means a customer will have more room to make mistake or be paralyzed about what to do. 

PS: If you think we must have the ability to have different provider options for inbound **and** outbound, I'll support you; mostly looking to understand what I'm missing.

Reply:
>PS: If you think we must have the ability to have different provider options for inbound and outbound, I'll support you; mostly looking to understand what I'm missing.

Yes, that's the reason why we should keep this implementation as is, because we never know if customers want to use the custom handler for inbound, outbound or both.

Reply:
FYI I chose not to use the `kwargs` value here because it is already being used to pass a custom set of arguments to the `handler` method. We wouldn't have a way to distinguish between arguments the user wants to pass to the json schema validation provider and arguments the user wants to pass to the `handler` method.